lib/thesis/models/page.ex

```diff
@@ -13,9 +15,27 @@ defmodule Thesis.Page do
   schema "thesis_pages" do
     field :slug, :string
     field :title, :string
+    field :template, :string
+    field :redirect_url, :string
     field :description, :string
 
     timestamps
   end
 
+  @doc """
+  Returns whether the page redirects to another page.
+
+  Examples:
+
+      iex> Thesis.Page.redirected?(%Thesis.Page{slug: "", redirect_url: nil})
+      false
+      iex> Thesis.Page.redirected?(%Thesis.Page{slug: "", redirect_url: ""})
+      false
+      iex> Thesis.Page.redirected?(%Thesis.Page{slug: "", redirect_url: "/asdf"})
+      true
+  """
+  def redirected?(nil), do: false
+  def redirected?(%Thesis.Page{redirect_url: url}) when url in [nil, ""], do: false
+  def redirected?(_), do: true
```

Comment:
Small function that tells us if this page has been redirected elsewhere. `nil`, `""`, or an unset (`nil`) page result in `false`, all else result in `true`.